internal/provider/provider.go

```diff
@@ -323,25 +323,27 @@ func New(ctx context.Context) (*schema.Provider, error) {
 			interceptors := interceptorItems{}
 
 			if v.Tags != nil {
-				// The resource has opted in to transparent tagging.
-				// Ensure that the schema look OK.
-				if v, ok := r.Schema[names.AttrTags]; ok {
-					if v.Computed {
-						errs = multierror.Append(errs, fmt.Errorf("`%s` attribute cannot be Computed: %s", names.AttrTags, typeName))
+				if r.SchemaFunc == nil {
```

Comment:
Can the schema be loaded from `r.SchemaFunc` to still complete this check in this case?

Reply:
Yes, good idea; the value will be temporary and gc-ed away.

Reply:
We can use the new `schema.Resource.SchemaMap()` function.